ts/features/fci/screens/valid/FciQtspClausesScreen.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from "react";
+import { SafeAreaView, FlatList, StyleSheet, View } from "react-native";
+import { useSelector } from "react-redux";
+import { constNull } from "fp-ts/lib/function";
+import { useNavigation } from "@react-navigation/native";
+import { H1 } from "../../../../components/core/typography/H1";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import I18n from "../../../../i18n";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import {
+  fciQtspClausesSelector,
+  fciQtspPrivacyTextSelector,
+  fciQtspPrivacyUrlSelector
+} from "../../store/reducers/fciQtspClauses";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import { useFciAbortSignatureFlow } from "../../hooks/useFciAbortSignatureFlow";
+import ItemSeparatorComponent from "../../../../components/ItemSeparatorComponent";
+import customVariables from "../../../../theme/variables";
+import QtspClauseListItem from "../../components/QtspClauseListItem";
+import { FCI_ROUTES } from "../../navigation/routes";
+import { H4 } from "../../../../components/core/typography/H4";
+import { Link } from "../../../../components/core/typography/Link";
+
+const styles = StyleSheet.create({
+  paddingText: {
+    paddingLeft: 4
+  }
+});
+const FciQtspClausesScreen = () => {
+  const [isClausesChecked, setIsClausesChecked] = React.useState(0);
```

Comment:
"is" prefix is used for boolean values. try to find a name that express a quantity/number.